addons/html_editor/tests/test_controller.py

```diff
@@ -205,3 +205,17 @@ def test_05_internal_link_preview(self):
         )
         self.assertEqual(200, response_not_record.status_code)
         self.assertTrue('other_error_msg' in response_not_record.text)
+
+        # test with URL path format `odoo/<model>/<record_id>`
```

Comment:
@walidsahli ,  can you add a test for abstract model here to make sure he can't access it